pack/rpm.mk

```diff
@@ -20,7 +20,7 @@ endif
 
 # Usual 'Release' RPM spec directive value is 1%{?dist}, where
 # where 1 is $(RELEASE) value and %{dist} is like .el8 or .fc31.
-RPMRELEASE := $(RELEASE)%{dist}
+RPMRELEASE := $(RELEASE)%{?dist}
```

Comment:
Same here. It is not clear why the change is needed. I would appreciate some explanation in the commit message/pull request description.

At least, I understand it as follows.

The given OS doesn't define the `dist` macro. It is unclear, whether a suffix like `.el7` or `.fc35` exists for packages provided by the OS repositories. If so, we should add it.

Reply:
Fixed (added separate section that follows the approach used in openSUSE, but using prefix 'at' instead of 'lp')

Reply:
Regarding 'at' prefix: it's a kind of heuristic. I just checked several packages on MosOS Arbat 15.4 and found out that many of them have 'at154' in their names in the same way as 'lp150/lp151/lp152' in openSUSE of versions 15.0, 15.1, 15.2.

Reply:
Thanks!